misc/nanovgutils/src/nanovgcontext.cpp

```diff
@@ -167,33 +171,37 @@ void NanoVGContext::textBox(const ivec2& coordinates, float textBoxWidth, const
     }
 }
 
-int NanoVGContext::createFont(const std::string& name, std::string filename) {
-    auto res = nvgCreateFont(activeNanoVGContext_, name.c_str(), filename.c_str());
-    if (res == -1) {
-        LogWarn("Failed to load font " << name << " from file: " << filename);
-    }
-    return res;
+int NanoVGContext::createFont(const std::string& name, const std::string& filename) {
+    return nvgCreateFont(activeNanoVGContext_, name.c_str(), filename.c_str());
 }
 
 void NanoVGContext::strokeWidth(float width) { nvgStrokeWidth(activeNanoVGContext_, width); }
 
 vec4 NanoVGContext::textBounds(const ivec2& position, const std::string& string) {
     float bounds[4];
     nvgTextBounds(activeNanoVGContext_, static_cast<float>(position.x),
-                  static_cast<float>(position.y), string.c_str(),
-                  &string.c_str()[string.size() - 1], bounds);
+                  static_cast<float>(position.y), string.c_str(), nullptr, bounds);
```

Comment:
You could use glm::value_ptr(bounds) to pass the pointer in here and make bound a vec4 to begin with

Reply:
There is probably a lot of those tweaks one can do in this file, this has been like that since before we moved it to this repo